bundles/org.openhab.core.automation.rest/src/main/java/org/openhab/core/automation/rest/internal/RuleResource.java

```diff
@@ -339,6 +343,22 @@ public Response getTriggers(@PathParam("ruleUID") @Parameter(description = "rule
         }
     }
 
+    @GET
+    @Path("/simulation")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "getRuleSimulation", summary = "Simulates the executions for the rules within the schedule.", responses = {
```

Comment:
Only a guess - I leave it up to you to change it. The method returns a list of rule simulations filtered by tag "Schedule". The description is not 100% clear and users might be confused. So we either could clarify it by defining an own path for "Scheduler" or add some more information in the description.

e.g.

```suggestion
    @Path("/scheduler/simulations")
    @Produces(MediaType.APPLICATION_JSON)
    @Operation(operationId = "getSchedulerRuleSimulations", summary = "Simulates the executions of rules filtered by tag 'Schedule'.", responses = {
```

Would it make sense to add the "tags" parameter?

Reply:
I've changed the rest endpoint according to your suggestion.
To allow filtering in simulation may be an good idea but would require additional work in web ui, so i would add the parameter when its needed.